src/index.js

```diff
@@ -36,6 +42,14 @@ try {
 		},
 	});
 
+	const { TRANSACTION_TYPES } = config.constants;
+	app.registerTransaction(TRANSACTION_TYPES.DAPP, DappTransaction);
+	app.registerTransaction(TRANSACTION_TYPES.IN_TRANSFER, InTransferTransaction);
```

Comment:
No matcher for now?

Reply:
@lsilvs created separate issue for this
https://github.com/LiskHQ/lisk/issues/3251